collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1912,7 +1912,7 @@ protected void onPause() {
         if (!formSaveViewModel.isSaving()) {
             if (currentView != null && formController != null
                     && formController.currentPromptIsQuestion()) {
-                formEntryViewModel.updateAnswersForScreen(getAnswers());
```

Comment:
We'd created this method and then DRYed up all the screen save code in `FormEntryActivity` not realizing that some screen saves flushed the audit log and others didn't.

Reply:
Yup it's very confusing right now. I didn't want to do too much restructuring for the hotfix though - we can look at that in the `master` fix.